example/.env.profile

```diff
@@ -0,0 +1,8 @@
+# This file is a template for defining the environment variables
+# Set the application config values here
+
+YOTI_APP_ID=388f7483-080e-48c5-b2fb-668783767d2c
```

Comment:
Don't put actual values here. I prefer naming these files `.env.example`, but it's up to you.

Reply:
That file shouldn't have been pushed - it got mixed up with the commits - I will remove it.